content/api/commands/invoke.md

```diff
@@ -11,6 +11,15 @@ subject, use [`.its()`](/api/commands/its).
 
 </Alert>
 
+<Alert type="warn">
+
+If you chain further methods off of `.invoke()`, it will be called multiple
+times. If the method is not idempotent, treat `.invoke()` like a command, and
```

Comment:
This isn't to avoid detached DOM errors - it's to avoid calling the method more than once. Updating the line to be more clear:

```
If you chain further methods off of `.invoke()`, it will be called multiple
times. If you only want the method to be called once, treat `.invoke()` like a
command, and consider it unsafe to chain further methods.
```